TombEngine/Game/Lara/lara_fire.cpp

```diff
@@ -33,23 +33,46 @@
 using namespace TEN::Entities::Generic;
 using namespace TEN::Input;
 using namespace TEN::Math;
+using std::pair;
```

Comment:
I am generally against shortening `std` types, because seeing `std::pair` in one place and `pair` in another can confuse someone.

Reply:
Oh, I have a habit of doing this. I can avoid it from now on and gradually revert wherever it's been done already.

Reply:
Yes please, generally shortening `std` types/methods is considered a bad practice, if I recall.